defaults/main.yml

```diff
@@ -1,10 +1,15 @@
 ---
-
 fits_download_dir: /usr/local/src
-fits_version: 1.1.1
-fits_ws_version: 1.1.3
+fits_version: 1.4.1
+fits_ws_version: 1.2.0
 fits_install_root: /opt
 fits_install_symlink: /opt/fits
 fits_user: tomcat8
 fits_group: tomcat8
 fits_ws: yes
+crayfits_home: /var/www/html
+fits_src_url: "https://github.com/harvard-lts/fits/releases/download/1.4.1/fits-{{ fits_version }}.zip"
```

Comment:
`https://github.com/harvard-lts/fits/releases/download/1.4.1/fits-{{ fits_version }}.zip` should be `https://github.com/harvard-lts/fits/releases/download/{{ fits_version }}/fits-{{ fits_version }}.zip`. No sense hardcoding one and not the other.